src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <div className="Person">
+      <p className="Person__name">
+        {`My name is ${name}`}
+      </p>
+      {age && (
```

Comment:
if `age === 0` then the `0` will be displayed before the `p` tag

```suggestion
      {age !== undefined && (
```